crates/mun_memory/tests/diff/structs.rs

```diff
@@ -348,16 +358,25 @@ fn cast_field() {
         vec![Diff::Edit {
             diff: vec![
                 FieldDiff::Edit {
-                    index: 0,
-                    kind: FieldEditKind::ConvertType,
+                    old_type: i64::type_info().clone(),
+                    new_type: f64::type_info().clone(),
+                    old_index: None,
```

Comment:
Does an `old_index` of `None` mean the `old_index` is actually the `new_index`?

Reply:
Yes, it means it wasn't moved